### PR TITLE
Add Java E-Book Programing

### PR DESCRIPTION
**Pull request by berviantoleo**

### CODE REVIEW

**Comment by berviantoleo** [comment]
I don't know why my code false. My code only at my first commit.

File: free-programming-books-id.md [hunk 1]
```diff
@@ -20,25 +21,30 @@
 * [Pro Git 1st Edition](https://git-scm.com/book/id/v1)
 
 
-### HTML CSS
+### HTML-CSS
```
Comment (vhf) [inline free-programming-books-id.md]: Is this necessary?

File: free-programming-books-id.md [hunk 2]
```diff
@@ -20,25 +21,30 @@
...
 
-* [Belajar HTML dan CSS](http://www.ariona.net/ebook-belajar-html-dan-css/)
+* [Belajar HTML dan CSS](http://www.ariona.net/ebook-belajar-html-dan-css)
 
 
+### Java
+
+* [Membuat Laporan di Java dengan Menggunakan IReport](https://app.box.com/s/xwxuw0jhe205mbyea2k5)
+* [Pemrograman Java oleh Rosihan Ari Yuana](https://app.box.com/s/mlw1pymsj9azm8trq8wwqwnk2m1asndi)
+
```
Comment (vhf) [inline free-programming-books-id.md]: [logs](https://travis-ci.org/vhf/free-programming-books/builds/168839400) say you're missing a line here.

File: free-programming-books-id.md [hunk 3]
```diff
@@ -20,25 +21,30 @@
...
 ### JavaScript
 
 * [Mengenal JavaScript](http://masputih.com/2013/01/ebook-gratis-mengenal-javascript)
-* [Otomatisasi dengan gulp.js](https://kristories.gitbooks.io/otomatisasi-dengan-gulp-js/content/)
+* [Otomatisasi dengan gulp.js](https://kristories.gitbooks.io/otomatisasi-dengan-gulp-js/content)
```
Comment (vhf) [inline free-programming-books-id.md]: Why did you modify this line?

Comment (vhf) [inline free-programming-books-id.md]: Same question for the following lines.

File: free-programming-books-id.md (hunk 1 above)
Comment (berviantoleo) [inline free-programming-books-id.md]: No, I will revert this change.

File: free-programming-books-id.md (hunk 3 above)
Comment (berviantoleo) [inline free-programming-books-id.md]: Sorry, I just try why error. I will revert this.

**Comment by vhf** [comment]
Thanks!

**Comment by vhf** [comment]
Could you please provide better links to these files? As a policy we don't accept files hosted on file sharing websites.

**Comment by berviantoleo** [comment]
I'm sorry. I think I couldn't find better links to these files. I will search to another books or tutorial. To change that. I haved place another tutorial page to there.

Thank you.

**Comment by eshellman** [comment]
Looks like the syntax linter wants a pdf indication

**Pull request merged**